tests/test.hs

```diff
@@ -373,7 +372,6 @@ loggingTestReporter = TestReporter [] $ \opts tree -> Just $ \smap -> do
                        "test, time(s), result"]
 
 
-    let dir = "tests" </> "logs" </> host ++ "-" ++ time
```

Comment:
The only use of `dir`, 5 lines further down, is commented out. This definition of `dir`, in turn, is the only place where `host` is used. That allowed me to comment out the definition of `host`, which is the only place that needed the `hostname` command, which allowed me to leave `hostname` out of `shell.nix`.

Reply:
Now removed instead of commented out.